diffrax/path.py

```diff
@@ -39,16 +39,26 @@ def evaluate(self, t0, t1=None, left=True):
     t0: Scalar = field(init=False, repr=False)
     t1: Scalar = field(init=False, repr=False)
 
+    def eval_levy(
+        self, t0: Scalar, t1: Optional[Scalar] = None, left: bool = True
+    ) -> LevyVal:
+        return self.evaluate(t0, t1, left, use_levy=True)
+
     @abc.abstractmethod
     def evaluate(
-        self, t0: Scalar, t1: Optional[Scalar] = None, left: bool = True
+        self,
+        t0: Scalar,
+        t1: Optional[Scalar] = None,
+        left: bool = True,
     ) -> PyTree:
         r"""Evaluate the path at any point in the interval $[t_0, t_1]$.
 
         **Arguments:**
 
         - `t0`: Any point in $[t_0, t_1]$ to evaluate the path at.
+
         - `t1`: If passed, then the increment from `t1` to `t0` is evaluated instead.
+
```

Comment:
Spurious change?

Reply:
This one was on purpose. It looked a bit funny in the docs. But yes, I can get rid of it.